docs/development/extensions-core/datasketches-tuple.md

```diff
@@ -207,3 +207,39 @@ Returns a human-readable summary of a given ArrayOfDoublesSketch. This is a stri
   "field"  : <post aggregator that refers to an ArrayOfDoublesSketch (fieldAccess or another post aggregator)>
 }
 ```
+
+
+### Constant ArrayOfDoublesSketch 
+
+This post aggregator adds a Base64-encoded constant ArrayOfDoublesSketch value that you can use in other post aggregators.
+```json
+{
+  "type": "arrayOfDoublesSketchConstant",
+  "name": DESTINATION_COLUMN_NAME,
+  "value": CONSTANT_SKETCH_VALUE
+}
+```
+
+### Base64 output of ArrayOfDoublesSketch 
+
+This post aggregator outputs an ArrayOfDoublesSketch as a Base64-encoded string storing the constant tuple sketch value that you can use in other post aggregators. 
+
+```json
+{
+  "type": "arrayOfDoublesSketchToBase64String",
+  "name": DESTINATION_COLUMN_NAME,
+  "field": <post aggregator that refers to a ArrayOfDoublesSketch (fieldAccess or another post aggregator)>
+}
+```
+
+### Estimated metrics values for each column of ArrayOfDoublesSketch
+
+This post aggregator returns a list of estimated sum for each metric value from a given ArrayOfDoublesSketch. The result is _N_ double values, where _N_ is the number of double values kept in the sketch per key.
```

Comment:
```suggestion
For each key-value pair in the given ArrayOfDoublesSketch, this post aggregator estimates the sum of the values associated with the key. The post aggregator returns _N_ double values, where _N_ is the number of double values associated with each key.
```

Based on the example in https://github.com/apache/druid/pull/13819#discussion_r1115305144, if the sum is done for each key (row-wise), then I think we should say `_N_ is the number of keys in the ArrayOfDoublesSketch` but if the sum is done across different keys (column-wise), then the description for _N_ is fine as is.

Reply:
@vtlim : The sum is performed column wise.

Reply:
Thanks for clarifying @anshu-makkar. Suggested wording, if this is accurate:

`````suggestion
For the key-value pairs in the given ArrayOfDoublesSketch, this post aggregator estimates the sum for each set of values across the keys. For example, the post aggregator returns `{3.0, 8.0}` for the following key-value pairs:

```
Key_1, {1.0, 3.0}
Key_2, {2.0, 5.0}
```

The post aggregator returns _N_ double values, where _N_ is the number of values associated with each key.
`````